compiler/ztests/sql-timeval.yaml

```
@@ -0,0 +1,10 @@
+zed: "yield TIMESTAMP '2012-10-02 12:01:01', DATE '2012-10-02 12:01:01'"
```

Nit: No need to quote this.
```suggestion
zed: yield TIMESTAMP '2012-10-02 12:01:01', DATE '2012-10-02 12:01:01'
```